packages/gasket-nextjs/package.json

```diff
@@ -66,7 +68,6 @@
     "mock-require": "^3.0.3",
     "next": "^12.0.8",
     "nyc": "^15.1.0",
-    "react": "^18.0.0",
```

Comment:
I'd leave this in `devDependencies` and pair it with an entry in `peerDependencies`.

Reply:
Should I do the same for prop-types?